Web/Edubase.Services/Lookup/CachedLookupService.cs

```diff
@@ -94,147 +94,72 @@ public CachedLookupService(ILookupService lookupService, ICacheAccessor cacheAcc
                     {
                         displayName= ((DisplayAttribute) attributes[0]).Name;
                     }
-                    
+
                     return displayName;
                 })) }
             };
         }
 
         public async Task<IEnumerable<LookupDto>> LocalAuthorityGetAllAsync() => await AutoAsync(_lookupService.LocalAuthorityGetAllAsync);
-        public IEnumerable<LookupDto> LocalAuthorityGetAll() => Auto(_lookupService.LocalAuthorityGetAll);
```

Comment:
Note to self -- these lines are removed because the synchronous (`NotImplementedException`) methods were removed (commented out) from `LookupApiService.cs`

Reply:
I should have made a note of this